src/ray/rpc/worker/worker_client.h

```diff
@@ -37,19 +56,119 @@ class WorkerTaskClient {
   /// \return if the rpc call succeeds
   ray::Status AssignTask(const AssignTaskRequest &request,
                          const ClientCallback<AssignTaskReply> &callback) {
-    auto call = client_call_manager_
-                    .CreateCall<WorkerTaskService, AssignTaskRequest, AssignTaskReply>(
-                        *stub_, &WorkerTaskService::Stub::PrepareAsyncAssignTask, request,
-                        callback);
+    auto call =
+        client_call_manager_
+            .CreateCall<WorkerService, AssignTaskRequest, AssignTaskReply>(
+                *stub_, &WorkerService::Stub::PrepareAsyncAssignTask, request, callback);
     return call->GetStatus();
   }
 
+  /// Push a task.
+  ///
+  /// \param[in] request The request message.
+  /// \param[in] callback The callback function that handles reply.
+  /// \return if the rpc call succeeds
+  ray::Status DirectActorAssignTask(
```

Comment:
Can we put these methods in a .cc?

Reply:
Good idea but I tried doing this and got a bunch of thread annotation errors (which I think means the existing thread annotations aren't actually working), so I will leave this to @ericl.